### ISSUE TITLE
I should be able to drag and drop into apps using the switcher

### ISSUE THREAD

**Issue by aral** [markstory/gala-alt-tab-plus#9]
## Use case

I have something I want to open in one of my running apps.

1. I start dragging in the source app
2. I open the task switcher and drop onto the icon of the app I want as the target of my drag and drop

## What should happen

A drop should register on the target app

## What actually happens

Nothing. 

(I’m actually unable to even carry out Step 2 due to #8).

**Comment by markstory** [comment]
Drag and drop is not something I had considered supporting. Is that something that other icon based application switchers support?

**Comment by hanaral** [comment]
I’m pretty sure macOS supports the ability to switch apps and workspaces by hovering while DnD’ing afaik

**Comment by thomas-shirley** [comment]
Yep - you can drag and drop into the alt+tab switcher on MacOS. It's a brilliant trick. +1 vote for this.
